README.md

```diff
@@ -1 +1,69 @@
-# AwesomeBooksES6
\ No newline at end of file
+<!-- Feel Free to Add, Update, Delete Any Section you find needs so -->
+
+## AwesomeBooksES6
+
+> .This project is BookStore so the user can add books by just enter Book name and the author After the user add it,
+it already displayed in the top of the page and can remove it by remove button.
+
+<img src="Capture.PNG" width="70%" />
+
+## Built With
+
+- Languages: _**HTML, CSS, JavaScript**_
+- Frameworks: _**N/A**_
+- Technologies used: _**GIT, GITHUB, LINTERS**_
+
+## Additional tools
+ - Google fonts
+
+## Live Demo
+
+[See My project Live here]()
+

```

Comment:
- [ ] Everything looks good, let's just include some information for a user who would like to have a copy of the project and running it locally.